docs/fermi_hubbard/experiment_example.ipynb

```diff
@@ -1,12 +1,65 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "4fa0ce4484f8"
+   },
+   "source": [
+    "##### Copyright 2020 Google"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "cellView": "form",
+    "id": "906e07f6e562"
+   },
+   "outputs": [],
+   "source": [
+    "#@title Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "# you may not use this file except in compliance with the License.\n",
+    "# You may obtain a copy of the License at\n",
+    "#\n",
+    "# https://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "# Unless required by applicable law or agreed to in writing, software\n",
+    "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "# See the License for the specific language governing permissions and\n",
+    "# limitations under the License."
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {
     "id": "e6d108d2b320"
    },
    "source": [
-    "# Fermi-Hubbard Experiment Example"
+    "# Fermi-Hubbard experiment example"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "8d9dec56909c"
+   },
+   "source": [
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://www.example.org/cirq/experiments/fermi_hubbard/experiment_example\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />View on QuantumLib</a>\n",
```

Comment:
This points to a dead domain.

Reply:
This is true for all tutorials currently, e.g. [QAOA](https://github.com/quantumlib/ReCirq/blob/master/docs/qaoa/example_problems.ipynb) and [Cirq basics](https://github.com/quantumlib/Cirq/blob/master/docs/tutorials/basics.ipynb). I think the plan is to update the links once the devsite is public - do you know more @mpharrigan?

Reply:
Actually, the "View on QuantumLib" links do not show up on the devsite.

Reply:
cc @lamberta , but I think @rmlarose is correct: we'll update once we have something to point to!

Reply:
Correct. We'll need to do a link sweep post-devsite (also update the icons). But this first button won't display on the site (but will in GitHub and Colab)